sky/services/keyboard/ios/keyboard_service_impl.mm

```diff
@@ -7,11 +7,11 @@
 
 static inline UIKeyboardType ToUIKeyboardType(::keyboard::KeyboardType type) {
   switch (type) {
-    case ::keyboard::KEYBOARD_TYPE_TEXT:
+    case keyboard::KeyboardType::TEXT:
```

Comment:
nit: should be consistent about fully qualifying or not fully qualifying ::keyboard here. since line 8 does, i'd say these lines should too i.e. ::keyboard::KeyboardType::TEXT. maybe even use a local using statement to pull them into the namespace